test/functional/wallet_listsinceblock.py

```diff
@@ -448,6 +449,19 @@ def test_send_to_self(self):
         assert any(c["address"] == addr for c in coins)
         assert all(self.nodes[2].getaddressinfo(c["address"])["ischange"] for c in coins)
 
+    def test_label(self):
+        self.log.info("Test label")
+        new_addr = self.nodes[1].getnewaddress("new_addr", "bech32")
+
+        self.nodes[2].sendtoaddress(new_addr, "0.001")
+        block_hash = self.generate(self.nodes[2], 1)[0]
+
+        labels = ["new_addr", ""]
+        for label in labels:
+            new_addr_transactions = self.nodes[1].listsinceblock(label=label)["transactions"]
```

Comment:
Since you have to retouch because of the linter, I would suggest adding `assert new_addr_transactions` for regression.
```suggestion
            new_addr_transactions = self.nodes[1].listsinceblock(label=label)["transactions"]
            assert new_addr_transactions
```

Should be ready now, good job!